### PR TITLE
Add boosters for HI, PA, and VT, and fix AZ, TN, and OR

### PR DESCRIPTION
**Pull request by smcclure17**

### CODE REVIEW

File: can_tools/scrapers/official/TN/tn_vaccine.py [hunk 1]
```diff
@@ -1,44 +1,43 @@
 import pandas as pd
-import requests
```
Comment (smcclure17) [inline can_tools/scrapers/official/TN/tn_vaccine.py:2]: [Tennessee's dashboard ](https://www.tn.gov/health/cedep/ncov/covid-19-vaccine.html) seems to have gone offline, but they have a CSV source that's probably better to pull from anyway, so I've updated this to use it instead

File: can_tools/scrapers/official/PA/philadelhpia_vaccine.py [hunk 2]
```diff
@@ -23,23 +23,24 @@ class PhiladelphiaVaccine(TableauDashboard):
...
     def fetch(self) -> pd.DataFrame:
         # create a dict of the 2 dose type tables
         # which are titled "Residents Percentage First" and "... Full"
         return {
             dose_type: self.get_tableau_view(dose_type=dose_type)[
-                self.data_tableau_table.format(dose_type=table_name)
+                self.data_tableau_table.format(dose_type=dose_type)
             ]
-            for dose_type, table_name in {"First": "New", "Full": "Full"}.items()
+            for dose_type in ["New", "Full", "Booster"]
```
Comment (smcclure17) [inline can_tools/scrapers/official/PA/philadelhpia_vaccine.py:38]: They've seemed to have switched back from the changes in #402, so I've reverted the code here to match

**Review by mikelehen — APPROVED** [review]
Looks good with minor nit!

File: can_tools/scrapers/official/AZ/counties/maricopa_vaccine.py [hunk 3]
```diff
@@ -44,10 +44,9 @@ def _get_json(self, url: str) -> list:
 
         # if the page must re-direct more than once, find the next re-direct url
         # re-call fn until found end page
-        if len(raw_data) == 0:
+        if len(raw_data) <= 1:
```
[the remark this answers is not in the document]
Comment (smcclure17) [reply can_tools/scrapers/official/AZ/counties/maricopa_vaccine.py]: Yeah good flag. I've reworked this slightly in 688e271b3af2059c0c2ca7aa6c899e504dc5b0da to look for the `http-equiv="refresh"` in `_get_json` in order to not rely on a certain number of `<script>`s being present

[the remark this answers is not in the document]
Comment (mikelehen) [reply can_tools/scrapers/official/AZ/counties/maricopa_vaccine.py]: Awesome, thank you!

**Pull request merged**